package.json

```diff
@@ -41,28 +42,30 @@
   },
   "private": true,
   "scripts": {
-    "bootstrap": "yarn && npm run check-versions && lerna run prepublish",
+    "bootstrap": "npm-run-all -s install-local check-versions lerna-prepublish",
     "check-versions": "babel-node scripts/check-versions.js",
-    "format": "npm run format-packages && npm run format-cache-dir && npm run format-www && npm run format-examples && npm run format-scripts && npm run format-markdown",
+    "format": "npm-run-all -p format-packages format-cache-dir format-www format-examples format-scripts format-markdown",
     "format-cache-dir": "prettier-eslint --write \"packages/gatsby/cache-dir/*.js\"",
     "format-examples": "prettier-eslint --write \"examples/**/gatsby-node.js\" \"examples/**/gatsby-config.js\" \"examples/**/src/**/*.js\"",
     "format-packages": "prettier-eslint --write \"packages/*/src/**/*.js\"",
     "format-scripts": "prettier-eslint --write \"scripts/**/*.js\"",
     "format-markdown": "prettier --write \"**/*.md\" --semi",
     "format-www": "prettier-eslint --write \"www/*.js\" \"www/src/**/*.js\"",
+    "install-local": "yarn",
```

Comment:
What's this?

Reply:
You will get a `ERROR: Task not found: "install"` error if trying to do `npm-run-all install`